src/bootstrap/dist.rs

```diff
@@ -2222,7 +2218,6 @@ impl Step for HashSign {
         cmd.arg(builder.package_vers(&builder.release_num("miri")));
         cmd.arg(builder.package_vers(&builder.release_num("rustfmt")));
         cmd.arg(builder.llvm_tools_package_vers());
-        cmd.arg(builder.lldb_package_vers());
```

Comment:
This is the line I am least sure about. It looks like this is about rustup packages and thus we should remove lldb here though?

Reply:
Yes, this is correct but you'll also want to remove it from src/tools/build-manifest/src/main.rs -- look for `let lldb_release =` to start.

Reply:
Like this?